.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.7', '3.10']
```

Comment:
Might be a good idea to also test with 3.11rc2, like we're currently doing in our other repos.

Reply:
Maybe we can wait until 3.11 is released and bump it in another PR then? I want to use both the oldest and newest supported Python versions.